src/ui/Facet/Facet.ts

```diff
@@ -257,14 +257,14 @@ export class Facet extends Component {
       section: 'Sorting',
       depend: 'enableSettings',
       values: [
-        'Occurrences',
-        'Score',
-        'AlphaAscending',
-        'AlphaDescending',
-        'ComputedFieldAscending',
-        'ComputedFieldDescending',
-        'ChiSquare',
-        'NoSort'
+        'occurrences',
```

Comment:
Maybe just double check that the strings that appear in the UI when you configure the option to allow end users to modify the sort on facets are not affected by this.

Reply:
I checked and they use the `defaultValue` property on the option. I am not sure if this property is used anymore, but I chose to keep it to avoid any break changes.